moveit_ros/planning/moveit_cpp/include/moveit/moveit_cpp/plan_solutions.h

```diff
@@ -0,0 +1,85 @@
+/*********************************************************************
```

Comment:
Was this moved from somewhere else and Git considers it an add, or a mistakenly added file?

Reply:
I changed the file format to .h which is why this is observed.

Reply:
[5578fde](https://github.com/ros-planning/moveit2/pull/1546/commits/5578fde8e758d827d0c08145b28b51844d07a59f)